core/core_test.go

```diff
@@ -224,7 +224,7 @@ func TestTranscode(t *testing.T) {
 		t.Errorf("Wrong SeqNo assigned to broadcaster: %v", b1.SeqNo)
 	}
 
-	if Over1Pct(len(b1.Data), 155476) || Over1Pct(len(b2.Data), 208304) {
+	if Over1Pct(len(b1.Data), 65424) || Over1Pct(len(b2.Data), 81968) {
```

Comment:
The test segment is actually 8 seconds long, so we limit it to 4 seconds to avoid hitting the length checker. https://github.com/livepeer/go-livepeer/pull/351/files#diff-190abcb36ea22f1ea680c1169087b646R156

So here we're setting the expected values based on a segment length that's just slightly under 4 seconds.